### PR TITLE
improvement(cli,migrate): cleanup and update db commands

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

File: src/packages/cli/src/CLI.ts [hunk 1]
```diff
@@ -119,9 +117,13 @@ export class CLI implements Command {
                 init   Setup Prisma for your app
           introspect   Get the datamodel of your database
             generate   Generate artifacts (e.g. Prisma Client)
-              format   Formats your schema
+              studio   Open Prisma Studio
```
Comment (janpio) [inline src/packages/cli/src/CLI.ts:120]: Why this unrelated change?

File: src/packages/cli/src/CLI.ts [hunk 2]
```diff
@@ -154,29 +161,35 @@ export class CLI implements Command {
...
-
-      Save your changes into a migration
-      ${chalk.dim('$')} prisma migrate save --experimental
-
+      
       Browse your data
       ${chalk.dim('$')} prisma studio
+
+      Push the Prisma Schema state to the database
+      ${chalk.dim('$')} prisma db push --preview
```
Comment (janpio) [inline src/packages/cli/src/CLI.ts:190]: Flag is undocumented under "Flags"

File: src/packages/cli/src/CLI.ts [hunk 3]
```diff
@@ -119,9 +117,13 @@ export class CLI implements Command {
...
             generate   Generate artifacts (e.g. Prisma Client)
-              format   Formats your schema
+              studio   Open Prisma Studio
+              format   Format your schema
+                  db   Manage your database schema and lifecycle ${chalk.dim(
+                    '(preview)',
+                  )}
 
-    ${chalk.bold('Flags')}
+    ${chalk.bold('Flag')}
```
Comment (janpio) [inline src/packages/cli/src/CLI.ts:126]: Inconsistent to below

File: src/packages/cli/src/CLI.ts [hunk 4]
```diff
@@ -135,9 +137,14 @@ export class CLI implements Command {
 
       Generate artifacts (e.g. Prisma Client)
       ${chalk.dim('$')} prisma generate
+
+      Browse your data
+      ${chalk.dim('$')} prisma studio
+
+      Push the Prisma Schema state to the database
+      ${chalk.dim('$')} prisma db push --preview
```
Comment (janpio) [inline src/packages/cli/src/CLI.ts:145]: Undocumented flag

File: src/packages/cli/src/Format.ts [hunk 5]
```diff
@@ -14,28 +14,26 @@ import path from 'path'
...
  */
 export class Format implements Command {
   public static new(): Format {
     return new Format()
   }
 
-  // static help template
   private static help = format(`
-    Format a Prisma schema.
+    Format a Prisma Schema.
```
Comment (janpio) [inline src/packages/cli/src/Format.ts:25]: Docs spell it "Prisma schema"

File: src/packages/cli/src/Studio.ts [hunk 6]
```diff
@@ -87,7 +87,7 @@ export class Studio implements Command {
       console.warn(
         `${chalk.yellow(
           'warn',
-        )} --experimental is no longer required for this command`,
+        )} --experimental is no longer required for this command as Studio now Generally Available.`,
```
Comment (janpio) [inline src/packages/cli/src/Studio.ts:90]: grammar

File: src/packages/cli/src/CLI.ts (hunk 1 above)
Comment (Jolg42) [reply src/packages/cli/src/CLI.ts:120]: You mean adding Studio? We forgot to do that when we switched Studio to GA 🤷‍♂️

File: src/packages/cli/src/Format.ts (hunk 5 above)
Comment (Jolg42) [reply src/packages/cli/src/Format.ts:25]: Ok I can fix that everywhere then

**Review by timsuchanek — APPROVED** [review]
LGTM

File: src/packages/cli/src/CLI.ts [hunk 7]
```diff
@@ -119,11 +129,16 @@ export class CLI implements Command {
                 init   Setup Prisma for your app
           introspect   Get the datamodel of your database
             generate   Generate artifacts (e.g. Prisma Client)
-              format   Formats your schema
+              studio   Open Prisma Studio
+              format   Format your schema
+                  db   Manage your database schema and lifecycle ${chalk.dim(
+                    '(preview)',
+                  )}
```
Comment (janpio) [inline src/packages/cli/src/CLI.ts:136]: `db` is not GA, so should only be visible when run with `--preview` (similar to `migrate` and `--experimental`)

File: src/packages/cli/src/CLI.ts [hunk 8]
```diff
@@ -135,9 +150,14 @@ export class CLI implements Command {
 
       Generate artifacts (e.g. Prisma Client)
       ${chalk.dim('$')} prisma generate
+
+      Browse your data
+      ${chalk.dim('$')} prisma studio
+
+      Push the Prisma schema state to the database
+      ${chalk.dim('$')} prisma db push --preview
```
Comment (janpio) [inline src/packages/cli/src/CLI.ts:158]: Same as https://github.com/prisma/prisma/pull/4027/files#r511902014

Comment (Jolg42) [reply src/packages/cli/src/CLI.ts:158]: As discussed, product decided to not hide commands in the Preview state

File: src/packages/cli/src/CLI.ts (hunk 7 above)
Comment (Jolg42) [reply src/packages/cli/src/CLI.ts:136]: As discussed, product decided to not hide commands in the Preview state
